src/backend/distributed/operations/repair_shards.c

```diff
@@ -932,12 +940,25 @@ CopyShardTablesViaBlockWrites(List *shardIntervalList, char *sourceNodeName,
 	ShardInterval *shardInterval = NULL;
 	foreach_ptr(shardInterval, shardIntervalList)
 	{
-		bool includeDataCopy = !PartitionedTable(shardInterval->relationId);
-
-		List *ddlCommandList = CopyShardCommandList(shardInterval, sourceNodeName,
-													sourceNodePort, includeDataCopy);
+		/*
+		 * For each shard we first create the shard table in a first separate
```

Comment:
nit; twice the word first makes the line funny to read

```suggestion
		 * For each shard we first create the shard table in a separate
```